lib/shared/puppet_forge/v3/module.rb

```diff
@@ -0,0 +1,31 @@
+require 'shared/puppet_forge/module_name'
+require 'shared/puppet_forge/connection'
+require 'shared/puppet_forge/v3/module_release'
+
+module PuppetForge
+  module V3
+    # Represents metadata for a single Forge module and provides access to
+    # the releases of a module.
+    class Module
+
+      include Connection
```

Comment:
Through the power of Ruby and dynamic scoping we don't _need_ it, but we shouldn't rely on said dynamic scoping. Will fix.